tests/src/JIT/Performance/CodeQuality/BenchI/XposMatrix/XposMatrix.cs

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
I think it makes sense to categorize these under tests/src/JIT given they're very focused.